src/scss/_brand.scss

```diff
@@ -71,9 +71,9 @@
 ));
 
 @if oBrandGetCurrentBrand() == 'master' {
-    $o-typography-sans: oFontsGetFontFamilyWithFallbacks(MetricWeb) !default!global;
```

Comment:
In Sass variables are scoped within `if` statements so we need the `!global` here. The variables are pre-defined in the global scope in [variables.scss](https://github.com/Financial-Times/o-typography/blob/master/src/scss/_variables.scss#L21), but that's [included after](https://github.com/Financial-Times/o-typography/blob/master/main.scss#L7) brand.scss. Dart Sass doesn't like that they aren't already defined in the global scope. 

We haven't added it to the spec but we've been consistently including `brand.scss` before `variables.scss` so Sass variables may be [set depending on the brand](https://github.com/Financial-Times/o-header-services/blob/master/src/scss/_variables.scss#L12).

It's not ideal but I'd suggest moving these [three variables](https://github.com/Financial-Times/o-typography/blob/master/src/scss/_variables.scss#L21) from `variables.scss` to the top of `brand.scss`. In the next `o-typography` major we can remove them in favour of a getter function along the lines of [_oTypographyFontFamilyForType](https://github.com/Financial-Times/o-typography/blob/master/src/scss/_functions.scss#L77).